ta/os_test/os_test.c

```diff
@@ -635,10 +635,9 @@ static TEE_Result test_time(void)
 	printf("TA time %u.%03u\n", (unsigned int)t.seconds,
 	       (unsigned int)t.millis);
 
-	if (t.seconds > sys_t.seconds) {
-		EMSG("Unexpected wrapped time %u.%03u (sys_t %u.%03u)\n",
-		     (unsigned int)t.seconds, (unsigned int)t.millis,
-		     (unsigned int)sys_t.seconds, (unsigned int)sys_t.millis);
+	if (t.seconds > 1) {
```

Comment:
In case of high CPU load, I think the test can legitimately reach a bit more than 1sec. So using 2 here is safer IMO.

Reply:
The test will pass with TA time up to 1s999ms.
You suggest doing `if (t.seconds > 2) { error } ` for a TA time up to 2s999ms ?

Reply:
You're right. Fine as is.